wordpress-mobile/WordPress-iOS
# Update wpxmlrpc to version 0.8.3

Updates the wpxmlrpc to avoid some explicit cast on swift code.

To test:
 - Run the app
 - Login using an self hosted site (non jetpack)
 - Check if the app is working correctly for that site (view posts, pages and write a post with an image).


Needs review: @koke
